authors/settings/dev.py

```diff
@@ -0,0 +1,19 @@
+import os
+from authors.settings.base import *
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+# Database
+# https://docs.djangoproject.com/en/1.11/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
```

Comment:
Change the environment variables to capital letters

Reply:
Environment variables have been edited to capital letters